package.json

```diff
@@ -45,7 +45,7 @@
     "express": "^4.13.4",
     "geojson-validation": "^0.1.6",
     "mapbox-gl": "^0.19.0",
-    "mapbox-gl-js-mock": "^0.20.0",
+    "mapbox-gl-js-mock": "^0.21.0",
```

Comment:
yea, I just looked at the README and realized that I didn't push the draft that talked about versioning. Sorry.